downloader/artifact_downloader_test.go

```diff
@@ -140,12 +140,11 @@ func Test_DownloadAndSaveArtifacts_RetriesFailingDownload(t *testing.T) {
 
 	assert.NoError(t, err)
 	// Total request count is:
-	// 1 + 2 * 5 = 11
+	// 1 + 5 = 6
```

Comment:
The got package handles small file sizes differently. It downloads <10Kb files with a concurrency of one.